docs/modules/customize/pages/oauth.adoc

```diff
@@ -30,4 +30,16 @@ ActiveSupport::Notifications.subscribe "decidim.user.omniauth_registration" do |
 end
 ----
 
+Note that, if the user is already registered, the `CreateOmniauthRegistration` command will not trigger the "decidim.user.omniauth_registration" event byt instead it will trigger the "decidim.user.omniauth_login" event, which has the same payload as the registration event.
```

Comment:
```suggestion
Note that, if the user is already registered, the `CreateOmniauthRegistration` command will not trigger the "decidim.user.omniauth_registration" event but instead it will trigger the "decidim.user.omniauth_login" event, which has the same payload as the registration event.
```